migration/v0.25.x-v0.26.x/add-support-message.sh

```diff
@@ -25,5 +25,5 @@ elif [ -f "${CK8S_CONFIG_PATH}"/../"${sc_template}" ]; then
   export OPENSEARCH_VALUE
   yq4 -i '.welcomingDashboard.extraTextOpensearch = strenv(OPENSEARCH_VALUE) | .welcomingDashboard.extraTextOpensearch style="double"' "${sc_config}"
 else
-  printf "dashboard-support-string.yaml file not found"
+  printf "file not found"
```

Comment:
```suggestion
  printf "${CK8S_CONFIG_PATH}/${sc_template} or ${CK8S_CONFIG_PATH}/../${sc_template}: not found"
```

Reply:
shellcheck didn't let me use variables in the printf format string :shrug: but w/e did something similar